### PR TITLE
Node.d.ts: Add module v8

### PR DESCRIPTION
**Pull request by hriss95**
case 2. Improvement to existing type definition.
- Add module v8 according to NodeJS' documentation [here](https://nodejs.org/api/v8.html)

### CODE REVIEW

File: node/node.d.ts [hunk 1]
```diff
@@ -2629,3 +2629,9 @@ declare module "constants" {
 declare module "process" {
     export = process;
 }
+
+declare module "v8" {
+    export function getHeapStatistics() : {total_heap_size: number, total_heap_size_executable: number, total_physical_size: number, total_avaialble_size: number, used_heap_size: number, heap_size_limit: number};
+    export function getHeapSpaceStatistics(): {space_name: string, space_size: number, space_used_size: number, space_available_size: number, physical_space_size: number};
```
Comment (vvakame) [inline node/node.d.ts]: > The value returned is an array of objects containing the following properties

I think we should use array type?

Comment (hriss95) [inline node/node.d.ts]: Yes, missed that one, sorry! Do you think I should make an interface called `HeapSpaceInfo` and then return HeapSpaceInfo[] just to make things look clearer?

Comment (vvakame) [inline node/node.d.ts]: 👍

**Comment by vvakame** [comment]
LGTM. but conflict occurred. please resolve it.

**Comment by hriss95** [comment]
@vvakame done :)

**Pull request merged**